mache-rabbit/src/main/java/com/excelian/mache/events/integration/RabbitMQConfig.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Type for the Rabbit MQ config.
  */
-public class RabbitMqConfig {
+public class RabbitMQConfig {
 
```

Comment:
I was hoping it may apply the two commit renames individually but I suspect anyone that already has the code checked out will need to rename the file locally (sorry!)